rust/blockstore/src/positional_posting_list_value.rs

```diff
@@ -96,18 +96,18 @@ impl PositionalPostingListBuilder {
         self.doc_ids.contains(&doc_id)
     }
 
-    pub fn add_positions_for_doc_id(
+    pub fn add_position_for_doc_id(
         &mut self,
         doc_id: i32,
-        positions: Vec<i32>,
+        position: i32,
     ) -> Result<(), PositionalPostingListBuilderError> {
         if !self.doc_ids.contains(&doc_id) {
             return Err(PositionalPostingListBuilderError::DocIdDoesNotExist);
         }
 
         // Safe to unwrap here since this is called for >= 2nd time a token
         // exists in the document.
-        self.positions.get_mut(&doc_id).unwrap().extend(positions);
+        self.positions.get_mut(&doc_id).unwrap().push(position);
```

Comment:
gives us a slight perf bump since current usage only adds one position at a time

Reply:
I removed all this extravaganza in my PR